### PR TITLE
Update plugin settings to allow to release it from GitHub

### PR DESCRIPTION
**Pull request by aheritier**

### CODE REVIEW

**Comment by aheritier** [comment]
@reviewbybees please review these changes to allow me to release it

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+.project
+build
+
+# vim
+*~
+
+# OS X
+.DS_Store
+
+# JEnv
```
Comment (KostyaSha) [inline .gitignore]: Just import blocks from https://github.com/github/gitignore

File: pom.xml [hunk 2]
```diff
@@ -25,24 +25,12 @@
       <name>Thomas Spengler</name>
     </developer>
   </developers>
-  
-  <distributionManagement>
-    <repository>
-      <uniqueVersion>false</uniqueVersion>
```
Comment (KostyaSha) [inline pom.xml]: replace with jenkins urls

File: .gitignore (hunk 1 above)
Comment (aheritier) [inline .gitignore]: Fixed by latest commit (I'll squash it later)

File: pom.xml (hunk 2 above)
Comment (aheritier) [inline pom.xml]: What do you mean ? 
They are already defined in jenkins parent ? Do I miss something ?

Comment (KostyaSha) [inline pom.xml]: then ok and not sure why all plugins defining distributionManagment explicitly

File: pom.xml [hunk 3]
```diff
@@ -6,13 +6,13 @@
   <packaging>hpi</packaging>
   <name>Hudson Distributed Workspace Cleaner</name>
   <version>1.0.5-SNAPSHOT</version>
-  <url>http://wiki.hudson-ci.org/display/HUDSON/Hudson+Distributed+Workspace+Clean+plugin</url>  
+  <url>https://wiki.jenkins-ci.org/display/JENKINS/Hudson+Distributed+Workspace+Clean+plugin</url>
```
Comment (jtnord) [inline pom.xml]: OT: looks like this should be renamed :-)

File: pom.xml [hunk 4]
```diff
@@ -6,13 +6,13 @@
   <packaging>hpi</packaging>
   <name>Hudson Distributed Workspace Cleaner</name>
```
Comment (jtnord) [inline pom.xml]: OT: Jenkins

**Comment by daniel-beck** [comment]
@recena Please don't comment on commits.

@aheritier Wiki pages can be renamed at any time (and the old URL will link to the new URL when accessed), but the update center generator will not publish a plugin if its newest release does not point to an existing wiki page. There may be a slight timing issue but if you release and rename around the same time, the plugin will disappear for at most four hours.

Jenkins uses the artifactId to know which plugin is which, so changing that will result in duplication in the update center (unless we exclude the obsolete one) and people not able to update.

File: pom.xml (hunk 4 above)
Comment (daniel-beck) [inline pom.xml]: This can be changed at any time.

**Comment by jtnord** [comment]
Given this has been updated from the Hudson infrastructure to Jenkins infrastructure I think this should also remove the "Hudson" from the name in both the WIKI url and the plugin name.

however its not enough for me to vote bug, but also not happy to vote bee either.

**Comment by aheritier** [comment]
@daniel-beck @jtnord thanks for these feedbacks. Thus I didn't touch to the groupId/artifactId but I removed Hudson from the plugin name and wiki page URL. I'll rename the wiki page when I'll update it for the release to be sure to not remove the plugin from the UC for a long time

**Comment by jtnord** [comment]
:bee:

**Comment by recena** [comment]
@daniel-beck I'm sorry.

**Comment by recena** [comment]
:bee: By the way, @stephenc are including [this document](https://github.com/jenkinsci/purge-job-history-plugin/blob/master/CONTRIBUTING.md) in the plugins which he is mantainer. IMO, it is a great idea.

**Comment by aheritier** [comment]
I will add it @recena I like it. It could be great to find an easy way to share/reuse it

**Comment by KostyaSha** [comment]
@aheritier :-1: for stephenc contributing.md

**Comment by aheritier** [comment]
@KostyaSha in any case I won't do it now. I'm focusing on releasing this plugin

**Comment by KostyaSha** [comment]
:shipit:

**Comment by aheritier** [comment]
@reviewbybees done

**Pull request merged**